### PR TITLE
[13.0] [MIG] purchase_packaging

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**

### CODE REVIEW

File: purchase_packaging/models/stock_warehouse_orderpoint.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        # Let's change the minimum rule to 5 units.
+        # The standard subtract_procurements_from_orderpoints will return 4
+        # and Odoo will create a procurement for 1 unit which will trigger a
+        # purchase of 12 due to the multiple. So the original purchase will
+        # be increased to 24 units which is wrong.
+        # This override will return 12 and no additionnal procurement will be
+        # created
+        res = super()._quantity_in_progress()
+        for orderpoint in self:
+            domain = [('orderpoint_id', '=', orderpoint.id),
```
Comment (rousseldenis) [inline purchase_packaging/models/stock_warehouse_orderpoint.py]: Do the search before the loop with orderpoint.ids and build dict per orderpoint.

Comment (ThomasBinsfeld) [reply purchase_packaging/models/stock_warehouse_orderpoint.py]: Done

**Comment by rousseldenis** [comment]
@ThomasBinsfeld This one is located in purchase-workflow now.

**Comment by sbejaoui** [comment]
> @ThomasBinsfeld This one is located in purchase-workflow now.

I created a PR to move this one to purchase-workflow https://github.com/OCA/purchase-workflow/pull/1190

this one can be closed.

**Pull request closed without merging**